src/renderer/components/TimelineSpace/Contents/Cards/Toot.vue

```diff
@@ -593,9 +597,14 @@ export default {
         }
       }
 
-      .reblogger-name {
+      .reblogger-name /deep/ {
```

Comment:
__[stylelint]__ <sub>posted by Sider</sub>

Expected selector ".toot .detail .reblogger .reblogger-name /deep/" to come before selector ".toot .detail .toot-header .user .display-name /deep/" (no-descending-specificity)